docs/glossary.rst

```diff
@@ -0,0 +1,30 @@
+Glossary
+========
+
+.. glossary::
+
+    reservoir simulator
+        Reservoir simulators such as OPM Flow or Eclipse. 
+
+    .DATA file
+        Inputs provided to reservoir simulators such as Eclipse or OPM Flow.
+        Usually a .DATA file pointing to other include files. One .DATA file
+        typically points to multiple include files. A data file is defined as
+        a **full** data file if ?...TODO
```

Comment:
don't' forget this TODO..

Reply:
Oops, and done, this is handled under the `deck` description: Changed the full/not full to complete/incomplete `deck`, so the `.DATA file` is no longer described as full/not full